README.md

```diff
@@ -25,6 +25,23 @@ We would be grateful if you could give us feedback on bugs and new feature reque
 ## Supported platforms
 This embedder supports x64 and Arm64 (aarch64, ARMv8) architectures on Linux which supports either Wayland backend or DRM backend.
 
+### Tested devices
+
+| Board/SoC | Vendor | OS | Display backend | Status |
+| :-------------: | :-------------: | :-------------: | :-------------: | :-------------: |
+| Desktop (x64) | Intel | Ubuntu18.04 | Wayland | :heavy_check_mark: |
+| Desktop (x64) | Intel | Ubuntu18.04 | DRM | :heavy_check_mark: |
+| [Jetson Nano](https://developer.nvidia.com/embedded/jetson-nano-developer-kit) | NVIDIA | JetPack 4.3 | Wayland | :heavy_check_mark: |
+| [Jetson Nano](https://developer.nvidia.com/embedded/jetson-nano-developer-kit) | NVIDIA | JetPack 4.3 | DRM | See: [#1](https://github.com/sony/flutter-embedded-linux/issues/1) |
+| [Raspberry Pi 4 Model B](https://www.raspberrypi.org/products/raspberry-pi-4-model-b/) | Raspberry Pi Foundation | Ubuntu 20.04 | Wayland | :heavy_check_mark: |
+| [Raspberry Pi 4 Model B](https://www.raspberrypi.org/products/raspberry-pi-4-model-b/) | Raspberry Pi Foundation | Ubuntu 20.04 | DRM | See: [#9](https://github.com/sony/flutter-embedded-linux/issues/9) |
```

Comment:
Since the desktop image is only released on `Ubuntu 20.10`, I tested Raspberry Pi on `Ubuntu 20.10`.
Should I test it on `Ubuntu 20.04`? 
See: https://ubuntu.com/download/raspberry-pi

Reply:
Sorry, it's my mistake.